### DISCUSSION TITLE
是否能够提供docker部署的版本

### DISCUSSION THREAD

**Discussion by sikenali** [he3-app/he3-feedback#146] [Q&A]
RT

**Comment by lyzhang1999** [comment]
感谢您的建议，已反馈给产品部门：https://github.com/he3-app/he3-feedback/issues/147 ，您可以关注此 issues。

**Comment by lyzhang1999** [comment]
另外想问一下您的具体使用场景是？docker 对 GUI 程序的支持可能比较差，如果要用 Windows 虚拟机的话则需要其他 Runtime 的支持，您是不方便在机器上安装工具吗？
